lib/trento/discovery/policies/cluster_policy.ex

```diff
@@ -278,6 +278,8 @@ defmodule Trento.Discovery.Policies.ClusterPolicy do
     end)
   end
 
+  defp parse_sbd_devices(_), do: []
```

Comment:
Enable empty sbd data coming, for future non sbd scenario.
PD: without this, we were having a nasty error